torch/onnx/_constants.py

```diff
@@ -4,8 +4,8 @@
 
 ONNX_BASE_OPSET = 9
 ONNX_MIN_OPSET = 7
-ONNX_MAX_OPSET = 20
-ONNX_TORCHSCRIPT_EXPORTER_MAX_OPSET = 20
+ONNX_MAX_OPSET = 23
+ONNX_TORCHSCRIPT_EXPORTER_MAX_OPSET = 23
```

Comment:
```suggestion
ONNX_TORCHSCRIPT_EXPORTER_MAX_OPSET = 17
```

Please keep this as 17, thanks! Support for opsets>17 is incomplete and is only contributed (the reduce* set of ops will be incorrect w/o updated implementations). At some point it was mistakenly set to 20.